package.json

```diff
@@ -32,15 +32,23 @@
   "dependencies": {
     "babel-polyfill": "^6.23.0",
     "babel-runtime": "^6.23.0",
+    "bem-classnames": "^1.0.7",
```

Comment:
What made you choose this module over the more commonly used classnames module? https://www.npmjs.com/package/classnames